src/platform/android.rs

```diff
@@ -19,6 +19,7 @@
 //!
 //! | winit |       ndk-glue               |
 //! | :---: | :--------------------------: |
+//! | 0.30  | `android-activity = "0.6"`   |
```

Comment:
Could you drop changes to this file, because I usually do them along the release bump, since there's no `0.30.0` version yet.

Reply:
Do you want me to drop this entirely, or insert a placeholder so that android-activity 0.6 is at least mentioned?

Reply:
just drop all the thing.